modules/react/button/stories/visual-testing/stories_Button.tsx

```diff
@@ -50,9 +51,4 @@ export const ButtonStates = () => (
   </StaticStates>
 );
 
-export const ButtonThemedStates = () => <ButtonStates />;
-ButtonThemedStates.parameters = {
-  canvasProviderDecorator: {
-    theme: customColorTheme,
-  },
-};
+export const ButtonThemedStates = () => <ButtonStates theme={{canvas: customColorTheme}} />;
```

Comment:
This change is subtle, but Docs mode doesn't handle story parameter changes like this. I thought it made more sense to pass the theme to the story instead. This way docs mode renders the theme story correctly.